NEWS.md

```diff
@@ -31,6 +31,9 @@ for human readability.
 
 #### Removed
 
+- Migrate neural network-based shock indicators to a new repository
+  [TrixiSmartShockFinder.jl](https://github.com/trixi-framework/TrixiSmartShockFinder.jl).
```

Comment:
This should be in a new section describing the upgrade from v0.5 to v0.6.

Reply:
3408d2ce4

Reply:
Thanks!